packages/web-react/src/components/Box/README.md

```diff
@@ -58,21 +64,21 @@ You can define background color using the `backgroundColor` prop.
-| ----------------- | ----------------------------------------------------------------- | ------- | -------- | ----------------------------- |
-| `backgroundColor` | [Background Color dictionary][dictionary-color]                   | -       | ✕        | Background color of the Box   |
-| `borderColor`     | [Border Color dictionary][dictionary-border-properities]          | -       | ✕        | Border color of the Box       |
-| `borderRadius`    | [Border Radius dictionary][dictionary-border-properities]         | -       | ✕        | Border radius of the Box      |
-| `borderStyle`     | [Border Style dictionary][dictionary-border-properities]          | `solid` | ✕        | Border style of the Box       |
-| `borderWidth`     | [Border Width dictionary][dictionary-border-properities]          | -       | ✕        | Border width of the Box       |
-| `elementType`     | `ElementType`                                                     | `div`   | ✕        | Type of element               |
-| `padding`         | \[`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | -       | ✕        | Padding of the Box            |
-| `paddingX`        | \[`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | -       | ✕        | Horizontal padding of the Box |
-| `paddingY`        | \[`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | -       | ✕        | Vertical padding of the Box   |
-| `paddingTop`      | \[`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | -       | ✕        | Padding top of the Box        |
-| `paddingRight`    | \[`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | -       | ✕        | Padding right of the Box      |
-| `paddingBottom`   | \[`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | -       | ✕        | Padding bottom of the Box     |
-| `paddingLeft`     | \[`SpaceToken` \| `Partial<Record<BreakpointToken, SpaceToken>>`] | -       | ✕        | Padding left of the Box       |
+| Name              | Type                                                                                            | Default | Required | Description                   |
+| ----------------- | ----------------------------------------------------------------------------------------------- | ------- | -------- | ----------------------------- |
+| `backgroundColor` | [Background Color dictionary][dictionary-color]                                                 | -       | ✕        | Background color of the Box   |
+| `borderColor`     | [Border Color dictionary][dictionary-border-properities]                                        | -       | ✕        | Border color of the Box       |
+| `borderRadius`    | \[`BorderRadiiDictionaryType` \| `Partial<Record<BreakpointToken, BorderRadiiDictionaryType>>`] | -       | ✕        | Border radius of the Box      |
```

Comment:
I was wondering if the radius and width should be prefixed with the token name as the spacing prop is.

Could someone explain to me why the spacing prop is set with `space-<value>` and radius or width with `<value>`?

Reply:
Solve this within [DS-1650](https://jira.almacareer.tech/browse/DS-1650)?

Reply:
Well, spacing uses space tokens, so it does make sense using values like `space-300`. And of course we do not want to allow every possible value.

However, if the same is applicable for the border radii, we should do this the same way, e.g. `radius-300` for example? Or maybe in the same way as Adobe Spectrum, but they have small -> large scale.

Reply:
And just a note for a future refactor: it would be nice to type this as `Responsive<BorderRadiusValue>` or something like that.

Reply:
https://jira.almacareer.tech/browse/DS-1819

Reply:
small → large it's a little bit short-sighted, because products can have a different radii scale.

I would rather use token names for values, but that should be separated issue. I've already write down a note into DS-1650.